source/community/consortium/mou/exhibit_a.md

```diff
@@ -0,0 +1,51 @@
+    * assembling demonstrations of IIIF solutions in action that illustrate its value to multiple levels of stakeholders, from users to senior leaders at major institutions; including demonstrations and metrics of impact;
+    * enhancing the IIIF.io website with accessible and useful information;
+    * conducting proactive outreach and dissemination.
+  * Develop and launch a IIIF interest group structure to foster the ongoing exchange of knowledge and progress towards work product in specific communities of interest (near term)
+  * Identify and cooperate with communities and initiatives with complementary members and objectives, including the Open Annotation and Web communities, standards bodies, technology providers, and cultural heritage organizations / services (such as Europeana and DPLA) (medium term)
+  * Apply IIIF to additional domains in like communities and content types (books, newspapers, journals, maps, music, museum objects, scrolls, archives). Seed the development of IIIF-based interest groups to catalyze further adoption, gathering of use cases, exposing more content, and attracting more software & solution providers to the Framework (medium term)
+  * Identify a host institution to help support IIIF activities with financial, organizational, administrative and other activities (medium term)
+  * Organize a more structured and sustainable community infrastructure to support and advance IIIF over the long term (ongoing)
+
+## API Specification
+
+  * Authentication / Authorization: extend the framework to support interoperability for access-controlled resources (near term)
+  * Search within an object; define strategies--including API extensions and reference implementations--for supporting textual search within a resource--such as of full text or text-based annotations, effectively extending IIIF support to books and newspapers (among other resources) (near term)
+  * Support and patterns for conveying attribution and licensing via IIIF (near term)
+  * Discovery of IIIF-compatible resources: enabling the ability to find interoperable, IIIF-based materials from the open web for local use (near term).
+  * Support for full management (create, edit, delete--not just read/view) via APIs for IIIF resources (manifests, images & annotations) (medium term)
+  * Creation, management and visualization of 3rd party annotations relative to IIIF-based resources (medium term)
+  * Support for physical (museum) objects and collections (medium term)
+  * Support for time-based media (audio/video) (long term)
+  * Updates to image and presentation APIs based on community adoption and feedback (ongoing)
+  * Investigate and instantiate a method or methods of assessing and reporting use of resources delivered by IIIF APIs from individual holding institutions 
+
+## Utilities & Adoption Aids
+
+Generally make IIIF easy to use and deploy, including through: 
```

Comment:
Is this already ratified text? Otherwise I'd drop "Generally".

Also "...including through: ... Gather" and "...including through: ... Develop" make no sense. In fact, "including through" doesn't make sense as a lead in for most of the clauses in the list.